src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -71,183 +88,184 @@ const SliderRow: React.FC<SliderRowProps> = ({ label, vals, valsReadout = vals,
   );
 };
 
-interface AxisClipSlidersProps {
+type PlaySliderRowProps = {
+  label: string;
+  val: number;
+  max: number;
+  playing: boolean;
+  entireAxisLoaded?: boolean;
```

Comment:
This var name made me pause.  I'm assuming this is only true when in 3D mode?  If so, it probably deserves a comment because I think it is rare that it could happen in any other way.  Maybe it could be named "canPlayWithoutAsync" or something similar...

Reply:
`updateWhileSliding`? Probably better to at least describe it in terms of what it changes about the slider's behavior. It's also true of the slice slider when looking down X or Y.